src/legacy/core_plugins/data/public/filter/filter_manager/lib/map_phrase.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { get, has } from 'lodash';
+import { PhrasesFilter } from '@kbn/es-query';
+import { SavedObjectNotFound } from '../../../../../../../plugins/kibana_utils/public';
+import { IndexPatterns, IndexPattern } from '../../../index_patterns';
+
+const TYPE = 'phrase';
```

Comment:
Nice to have, but I'm not sure if in the context of this PR.

I think we need to have a filter type enumeration \ mapping.
For example, see `src/legacy/core_plugins/data/public/filter/filter_bar/filter_editor/index.tsx` where the types are hardcoded as strings again.